syntax = "proto3";
package order;

service OrderService {
  rpc GetOrder (OrderRequest) returns (OrderReply);
  rpc CreateOrder (CreateOrderRequest) returns (OrderReply);
}

message OrderRequest {
  string id = 1;
}

message OrderReply {
  string id = 1;
  string user_id = 2;
  string product_name = 3;
  int32 quantity = 4;
}

message CreateOrderRequest {
  string user_id = 1;
  string product_name = 2;
  int32 quantity = 3;
}